static/docker-compose.yml

```diff
@@ -60,8 +60,8 @@ services:
       PG_RUNBOOKS_ADMIN_PASSWORD: ${PG_RUNBOOKS_ADMIN_PASSWORD:?err}
       PG_RUNBOOKS_ADMIN_USER: ${PG_RUNBOOKS_ADMIN_USER:?err}
       PG_RUNBOOKS_DB: ${PG_RUNBOOKS_DB:?err}
-      PG_TENANTS_WRITE_MODE: ${PG_TENANTS_WRITE_MODE:-couchbase_only}
-      PG_TENANTS_READ_MODE: ${PG_TENANTS_READ_MODE:-couchbase_only}
+      PG_TENANTS_WRITE_MODE: ${PG_TENANTS_WRITE_MODE}
+      PG_TENANTS_READ_MODE: ${PG_TENANTS_READ_MODE}
```

Comment:
```suggestion
      PG_TENANTS_WRITE_MODE: ${PG_TENANTS_WRITE_MODE:-}
      PG_TENANTS_READ_MODE: ${PG_TENANTS_READ_MODE:-}
```

Reply:
I see - this makes sense since env vars are always strings - there's no way to define an env var as "undefined"